cmake/modules/FindLibUSB.cmake

```diff
@@ -79,6 +79,7 @@ if(NOT LIBUSB_FOUND)
             file(DOWNLOAD
                 https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-${LIBUSB_WIN_VERSION}/libusb-${LIBUSB_WIN_VERSION}.7z/download
                 ${LIBUSB_WIN_ARCHIVE_PATH}
+                EXPECTED_MD5 750e64b45aca94fafbdff07171004d03
```

Comment:
Good idea, but what happens when a new version is released? The checksum will change, won't it? I don't want to make in very complicated, but can we retrieve it automatically? What do you think?

Reply:
Yes, when someone increases libusb version also should update MD5 accordingly.
But this prevents errors in archives.
Like this https://github.com/texane/stlink/issues/896#issuecomment-603877423 .

Reply:
Yes, no doubt - I did not question whether to have it or not...
I was only on about how to implement it, but referring to your mention in the main thread:
If it's only once per year an average we shall be fine and can do this manually.